crates/router/src/connector/bitpay/transformers.rs

```diff
@@ -284,4 +284,6 @@ pub enum WebhookEventType {
     Refunded,
     #[serde(rename = "invoice_manuallyNotified")]
     Resent,
+    #[serde(other)]
+    Unknown,
```

Comment:
Check where the webhook event types are being matched, and remove matching on `_` if it is happening.